Add Gallery page (#85)
kasenvr/vircadia-docs-sphinx: docs/source/explore/gallery.rst

#######
Gallery
#######

> Here are some pictures of past events and interface elements.

Author: The idea was to show more, but I don't necessarily want to show the Meeting Room in the Explore app 6 times and most other ui elements are to be changed in the near future.

> Here are some pictures of past events and Interface elements.

Reviewer: ```suggestion
This gallery showcases the Vircadia Interface and some past events and locations attended with it.
```

.. figure:: _images/gallery/el_papa_viejo.jpg
    :alt: Maker meeting 28th September 2020
    :class: inline

>     Maker meeting, 28th September 2020, on the "El Papa Viejo" (meaning "The Big Pope").

Reviewer: ```suggestion
    Maker Meeting, 28th September 2020, on the "El Papa Viejo" (meaning "The Old Pope").
```

Reviewer: Viejo means "old" instead of "big" from as far as I can find from internet resources.

.. figure:: _images/gallery/vircadia-snap-by-RyanLi-on-2020-06-12_14-28-47.jpg
    :alt: Startup Asia X 12th June 2020
    :class: inline

    Startup Lab Lagoon Koza booth at Startup Asia X, 12th June 2020.

.. figure:: _images/gallery/vircadia-snap-by-RyanLi-on-2020-06-12_14-52-39.jpg
    :alt: Startup Asia X 12th June 2020
    :class: inline

    Group photo at the end of Startup Asia X, 12th June 2020.

.. figure:: _images/gallery/vircadia-snap-by-RyanLi-on-2020-09-20_02-45-59.jpg

>     :alt: Developer meeting 20th September 2020

Reviewer: ```suggestion
    :alt: Developer Meeting 20th September 2020
```

    :class: inline

>     Dale Glass speaking at the Developer meeting, 20th September 2020.

Reviewer: ```suggestion
    Dale Glass speaking at the Developer Meeting, 20th September 2020.
```

.. figure:: _images/gallery/vircadia-snap-by-Revofire-on-2020-08-13_17-21-43.jpg

>     :alt: Community meeting 13th July 2020

Reviewer: ```suggestion
    :alt: Community Meeting 13th July 2020
```

    :class: inline

    Mannequin army, 13th July 2020.

.. figure:: _images/gallery/vircadia-snap-by-Revofire-on-2020-08-05_17-46-26.jpg
    :alt: Pictionary 5th July 2020
    :class: inline

    Pictionary in the old Hub, 5th July 2020.

.. figure:: _images/gallery/29th_september_interface_linux.jpg
    :alt: 29th September 2020
    :class: inline

>     Vircadia running on Linux Mint, 19.3 29th September 2020.

Reviewer: ```suggestion
    Vircadia running on Linux Mint 19.3, 29th September 2020.
```
